### PR TITLE
update logging sidecar template

### PR DESCRIPTION
**Pull request by pgvishnuram**
## Description

Initially we processed all message via single transform with remap , that was causing json errors in multiple components even-though it was not valid issue. 

The change added in this PR will filter logs based on components and process the logs , this has fixed parse error  for  webserver, scheduler and trigger components

## Related Issues

continuation from the below issue https://github.com/astronomer/issues/issues/4648

## Testing

Yet to update

## Merging

cherry-pick to 1.7, 1.6( not yet confirmed)

### CODE REVIEW

**Comment by vishwas-astro** [comment]
@pgvishnuram - is there a GH issue for this? If not, please create one.

File: templates/logging-sidecar-configmap.yaml
Comment (jedcunningham) [inline templates/logging-sidecar-configmap.yaml:51]: Does this approach also work for the workers log? I'm thinking we might need to be a little more specific with our condition?

Comment (pgvishnuram) [reply templates/logging-sidecar-configmap.yaml:51]: Yes - i have tested this change

Comment (pgvishnuram) [reply templates/logging-sidecar-configmap.yaml:51]: transform_webserver:
        type: filter
        inputs:
          - log_files
        condition: '.component == "worker"'

Comment (pgvishnuram) [reply templates/logging-sidecar-configmap.yaml:51]: with vrl
```
transform_worker:
        type: filter
        inputs:
          - transform_syslog
        condition: 
          type: "vrl"
          source: '.component == "worker"'
```

Comment (jedcunningham) [reply templates/logging-sidecar-configmap.yaml:51]: Had a chat with @pgvishnuram and both worker logs and task logs need to follow the 'task log' path here because they are both in json format.

File: templates/logging-sidecar-configmap.yaml
Comment (danielhoherd) [inline templates/logging-sidecar-configmap.yaml:36]:
```suggestion

```

File: templates/logging-sidecar-configmap.yaml
Comment (danielhoherd) [inline templates/logging-sidecar-configmap.yaml:53]:
```suggestion

```

File: templates/logging-sidecar-configmap.yaml
Comment (danielhoherd) [inline templates/logging-sidecar-configmap.yaml:77]:
```suggestion

```

**Review by danielhoherd — APPROVED** [review]

**Comment by vishwas-astro** [comment]
Can we get an approval from @jedcunningham and @danielhoherd before the merge please?

**Review by danielhoherd — APPROVED** [review]

**Pull request merged**
